__tests__/cache-save.test.ts

```diff
@@ -150,6 +152,23 @@ describe('run', () => {
       );
       expect(setFailedSpy).not.toHaveBeenCalled();
     });
+
+    it('should not save cache for npm', async () => {
```

Comment:
```suggestion
    it('should not save cache for pnpm', async () => {
```

I think it should be this?

Reply:
Good catch, fixed!